src/diagnostics/define_diagnostics.jl

```diff
@@ -189,36 +194,40 @@ function define_diagnostics!(land_model)
             compute_soil_water_liquid!(out, Y, p, t, land_model),
     )
 
+    # Infiltration
     add_diagnostic_variable!(
         short_name = "infil",
         long_name = "Infiltration",
         standard_name = "infiltration",
-        units = "m s^-1", # double check
+        units = "m s^-1",
         comments = "The flux of liquid water into the soil.",
```

Comment:
looks good. might say "volume flux of liquid water"

Reply:
hmm a flux is the amount of something going through a surface over some period of time. 
It is not always a volume (e.g., flux of photons), but for water, it usually is a volume? 
(it feels weird to me to say "volume flux"). But I agree (m s^-1) can be a bit unintuitive. I'll add (m^3 or water per m^2 of ground per second).